examples/game_of_life/measurement.py

```diff
@@ -24,15 +24,16 @@
 @measurement_service.configuration("width", nims.DataType.UInt32, 100)
 @measurement_service.configuration("height", nims.DataType.UInt32, 100)
 @measurement_service.configuration("update_interval", nims.DataType.UInt32, 100)
+@measurement_service.configuration("max_generations", nims.DataType.Int32, -1)
```

Comment:
Please add a constant like `INFINITE_GENERATIONS = -1` to document that -1 means infinite.

It could also make sense to change the check to `while max_generations == INFINITE_GENERATIONS or generation < max_generations:`